crates/types/src/services/executor.rs

```diff
@@ -379,6 +242,8 @@ pub enum Error {
     RelayerGivesIncorrectMessages,
     #[display(fmt = "Consensus parameters not found for version {_0}")]
     ConsensusParametersNotFound(ConsensusParametersVersion),
+    #[display(fmt = "The transaction is not in the block")]
+    L1TransactionNotInBlock,
```

Comment:
nit: should this be `DaTransactionNotInBlock`?

Reply:
This is being removed because we changed how the checks work.

But, in general, I tend to think of "DA" as being for fraud proving stuff, which is irrelevant from this perspective. What's relevant is that it's coming from the parent chain, so "L1" or "Relayed" are my chosen descriptors here.

Yes, our implementation will have "DA" and "L1" be the same, but the block producer/validator just needs txs from some source.